test/externalTests.sh

```diff
@@ -48,7 +50,7 @@ printTask "Running external tests..."
 "${REPO_ROOT}/test/externalTests/pool-together.sh" "$@"
 "${REPO_ROOT}/test/externalTests/perpetual-pools.sh" "$@"
 "${REPO_ROOT}/test/externalTests/uniswap.sh" "$@"
-"${REPO_ROOT}/test/externalTests/prb-math.sh" "$@"
+python "${REPO_ROOT}/test/externalTests/prb-math.py" --solc-binary-type "$BINARY_TYPE" --solc-binary-path "$BINARY_PATH"
```

Comment:
```suggestion
python3 "${REPO_ROOT}/test/externalTests/prb-math.py" --solc-binary-type "$BINARY_TYPE" --solc-binary-path "$BINARY_PATH"
```

Reply:
This script was removed in favor of https://github.com/ethereum/solidity/pull/14064/files#diff-a44410f6309c2dc53991630c52ef3bc24deaf40df2c3b33bde1dc858114152b9